src/renderer/src/routes/pages/BuyAndSellTokens/BuyTokensAccountsSide.tsx

```diff
@@ -0,0 +1,51 @@
+import { useLayoutEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useWalletsSelector } from '@renderer/hooks/useWalletSelector'
+import { IAccountState } from '@shared/@types/store'
+import { motion, useAnimate } from 'framer-motion'
+
+import { BuyTokensWalletItem } from './BuyTokensWalletItem'
+import { useBuyAndSellTokensContext } from './context'
+
+type TProps = {
+  account?: IAccountState
+}
+
+export const BuyTokensAccountsSide = ({ account }: TProps) => {
+  const { t } = useTranslation('pages', { keyPrefix: 'buyAndSellTokens.buyTokensAccountsSide' })
+  const { isAccountsSideOpened } = useBuyAndSellTokensContext()
+  const [scope, animate] = useAnimate()
+  const { walletsRef } = useWalletsSelector()
+  const wallets = account ? walletsRef.current.filter(wallet => wallet.id === account.idWallet) : walletsRef.current
+
+  useLayoutEffect(() => {
+    animate(
+      scope.current,
+      {
+        width: isAccountsSideOpened ? 364 : 0,
+        opacity: isAccountsSideOpened ? 1 : 0,
+        pointerEvents: isAccountsSideOpened ? 'auto' : 'none',
+      },
+      { type: 'spring', duration: 0.4 }
+    )
+  }, [animate, scope, isAccountsSideOpened])
+
+  return (
+    <motion.div
+      id="buy-and-sell-tokens-accounts-side"
+      aria-hidden={!isAccountsSideOpened}
+      ref={scope}
+      className="bg-asphalt/50 w-full h-full p-4 border-l border-gray-300/15 -mr-4 overflow-y-auto"
+    >
+      <h3 className="text-xs uppercase text-gray-300">{t('title')}</h3>
+
+      <ul className="flex flex-col gap-3 mt-3">
+        {wallets.map((wallet, index) => (
+          <li key={wallet.id} className="w-full">
+            <BuyTokensWalletItem isDefaultOpened={index === 0} wallet={wallet} />
+          </li>
+        ))}
```

Comment:
It doesn't make sense, you are rendering one accordion per wallet, whereas it should be an `Accordion.Root` with many `Accordion.Items`.

https://www.radix-ui.com/primitives/docs/components/accordion